apps/ui-layout/registry/components/drawer/motion-drawer.tsx

```
@@ -18,16 +18,16 @@ const data: ListItem[] = [
 function MotionDrawerExample() {
   return (
     <div>
-      <MotionDrawer direction='left' width={300} className='text-black'>
+      <MotionDrawer direction='left' width={300} className='text-foreground'>
         <nav className='space-y-4'>
           <h2 className='text-xl font-bold mb-4'>Menu</h2>
-          <a href='#' className='block p-2 hover:bg-neutral-100 rounded-sm'>
+          <a href='#' className='block p-2 hover:bg-muted rounded-sm'>
```

Biome (2.4.16)

[error] Provide a valid value for the attribute href.

(lint/a11y/useValidAnchor)